lib/modules/dosomething/dosomething_rogue/dosomething_rogue.module

```diff
@@ -71,14 +70,17 @@ function dosomething_rogue_send_reportback_to_rogue($values, $user = NULL) {
     'file' => isset($values['file']) ? $values['file'] : NULL,
     'caption' => isset($values['caption']) ? $values['caption'] : NULL,
     'status' => isset($values['status']) ? $values['status'] : 'pending',
-    'crop_x' => $values['crop_x'],
-    'crop_y' => $values['crop_y'],
-    'crop_width' => $values['crop_width'],
-    'crop_height' => $values['crop_height'],
-    'crop_rotate' => $values['crop_rotate'],
     'source' => isset($values['source']) ? $values['source'] : NULL,
   ];
 
+  if ($values['crop_x']) {
```

Comment:
since we are changing this code in multiple places it might make sense to pull this out into a helper function that prepares the item before sending to rouge, but you can do a follow up PR after merge if you want